Led.cpp

```diff
@@ -1,7 +1,33 @@
 #include "Led.h"
 
-Led::Led() {
+Led::Led() : pin(pin) {
 }
 
-Led::~Led() {
-}
\ No newline at end of file
+Led::Led(unsigned pin) : pin(pin) {
+    //	pinMode(pin, OUTPUT);
+}
+
+void Led::setBrightness(unsigned percent) {
+    //	analogWrite(led, precent);
+    brightness = percent;
```

Comment:
It would be best to convert `precent` here to the correct value range for when used by analogWrite